src/mtev_cluster.c

```diff
@@ -265,6 +359,8 @@ mtev_cluster_update_config(mtev_cluster_t *cluster, mtev_boolean create) {
     xmlSetProp(parent, (xmlChar *)"port", (xmlChar *)port);
     snprintf(period, sizeof(period), "%d", cluster->period);
     xmlSetProp(parent, (xmlChar *)"period", (xmlChar *)period);
+    snprintf(timeout_ms, sizeof(timeout_ms), "%d", cluster->timeout_ms);
+    xmlSetProp(parent, (xmlChar *)"timeout_ms", (xmlChar *)period);
```

Comment:
All times in libmtev (like period, and timeout) are millisecond, having just this one with a _ms ending breaks that consistency.  change to "timeout."

Also, I think that a "maturity" variable would be good, it can default to "timeout" but could be specified separately.  Only nodes that have seen last_context within "timeout" and have a boot time older than "maturity"